openlibrary/plugins/upstream/addbook.py

```diff
@@ -605,21 +631,25 @@ def _prevent_ocaid_deletion(self, edition):
             logger.warn("Attempt to change ocaid of %s from %r to %r.", self.edition.key, self.edition.get('ocaid'), ocaid)
             raise ValidationException("Changing Internet Archive ID is not allowed.")
 
-    def do_work_keys_match(self, i):
+    @staticmethod
+    def is_work_data_important(formdata):
         """
         Check if the form data's work matches the form data's edition's work.
```

Comment:
Check if the form data's work ID matches the form data's edition's work ID. If they don't match, ...